packages/Main/src/Layer/VpcLayer.js

```diff
@@ -0,0 +1,155 @@
+import * as THREE from 'three';
+import PointCloudNode from 'Core/PointCloudNode';
+import CopcNode from 'Core/CopcNode';
+import EntwinePointTileNode from 'Core/EntwinePointTileNode';
+import PointCloudLayer from 'Layer/PointCloudLayer';
+
+function markForDeletion(elt) {
+    if (elt.obj) {
+        elt.obj.visible = false;
+        if (__DEBUG__) {
+            if (elt.obj.boxHelper) {
+                elt.obj.boxHelper.visible = false;
+            }
+        }
+    }
+
+    if (!elt.notVisibleSince) {
+        elt.notVisibleSince = Date.now();
+        // Set .sse to an invalid value
+        elt.sse = -1;
+    }
+    for (const child of elt.children) {
+        markForDeletion(child);
+    }
+}
+
+function _intanciateNode(source) {
+    let root;
```

Comment:
rename method to `_instanciateRootNode` or rename this variable to `node`, depending on whether it is speicific to the root node or not.